authoringAssistant/src/main/java/explorableviz/transparenttext/QueryContext.java

```diff
@@ -169,9 +169,7 @@ public Optional<String> validate(String output) throws Exception {
         logger.info(STR."Validating output: \{output}");
         //Extract value from input query.text
         Optional<TextFragment> textFragment = paragraph.stream().filter(t -> t.getValue().contains("[REPLACE")).findFirst();
```

Comment:
We didn’t get a chance to discuss this properly the other day but it feels wrong to search for `[REPLACE` here given that it’s also the first thing that happens since `splitLiteral`.  I’ll merge the PR for now so as not to hold things up any further but let’s pick this up on Tuesday.